docs/resources/domain_credential.md

```diff
@@ -0,0 +1,46 @@
+---
+page_title: "Mailgun: mailgun_domain"
+---
+
+# mailgun\_domain_cedential
+
+Provides a Mailgun domain credential resource. This can be used to create and manage credential in domain of Mailgun.
+
+## Example Usage
+
+```hcl
+# Create a new Mailgun credential
+resource "mailgun_domain_credential" "foobar" {
+	domain = "toto.com"
+	email = "test@toto.com"
+	password = "supersecretpassword1234"
+	region = "us"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `domain` - (Required) The domain to add credential of Mailgun.
+* `email` - (Required) The email address to create.
+* `password` - (Required) Password for user authentication.
```

Comment:
When credential has been created, I run `terraform plan` and it discovered change on password field, but I didn't make any changes in my tf file. It's not natural behaviour of resource management. If there is no way to retrieve password via API, it has to be documented here with the snippet of how if could be handled e.g. by `ignore_changes`.

Reply:
In case of update with the same password Mailgun API returns: ` "message": "New password cannot be the same as old password."`, perhaps there is another API method for credentials validation? It could be used for discover the password changes.